packages/space-opera/src/components/mobile_view/mobile_view.ts

```diff
@@ -156,45 +158,27 @@ export class MobileView extends ConnectedLitElement {
     await this.triggerFetchLoop();
   }
 
-  async prepareGlbBlob(gltf: GltfModel) {
-    const glbBuffer = await gltf.packGlb();
-    return new Blob([glbBuffer], {type: 'model/gltf-binary'});
-  }
-
-  // Post new blob for scene-viewer
-  async postSceneViewerBlob(blob: Blob) {
-    const response = await fetch(this.modelViewerUrl, {
-      method: 'POST',
-      body: blob,
-    })
-    if (response.ok) {
-      console.log('Success:', response);
-    }
-    else {
-      throw new Error(`Failed to post: ${this.modelViewerUrl}`);
-    }
-  }
-
-  async newModel() {
+  // When the model is loaded, we make a post for this specific model for
+  // scene-viewer. Subsequently, everytime scene-viewer is opened, we send the
+  // POST again.
```

Comment:
Is it possible to just open a channel that will accept a fetch, instead of specifically posting and hoping it doesn't time out?